tests/test_sparse_array.py

```diff
@@ -710,6 +710,13 @@ def compare_with_sparse(full, sparse):
     assert np.array_equal(m_array[[False, True], 0][0], position_one)
 
 
+def test_keep_flyback(sparse_array_small):
```

Comment:
no test?

Reply:
Looks like I started the test but did not finish it.

Reply:
test added